azkaban-common/src/main/java/azkaban/executor/container/DummyEventListener.java

```diff
@@ -16,17 +16,16 @@
 package azkaban.executor.container;
 
 import azkaban.event.EventListener;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
+import org.apache.log4j.Logger;
 
 /**
- * This class is a no-op implementation of {@link EventListener}
+ * This class is a no-op implementation of {@link FlowStatusChangeEventListener}
  */
 public class DummyEventListener implements EventListener {
```

Comment:
This can be generic class. You can move it to the package where EventListener is and also no need to use logger of FlowStatusChangeEventListener class. Use its own class name.

Reply:
moved to azkaban/executor